packages/openapi/src/OpenAPIModule.ts

```diff
@@ -458,4 +472,11 @@ export class OpenAPIModule extends Module {
 
 		return jsonSchema;
 	}
+
+	private joinPaths(...args: Array<string>) {
+		return args
+			.join('/')
+			.replace(/\/+/g, '/') // replace multiple slashes with one
+			.replace(/\/$/, ''); // remove trailing slash
```

Comment:
thank you @juanjoseruiz, I forgot to apply the same revised logic to the OpenAPI module too.

Because OpenAPI and HttpServer are separate modules, I'd prefer to keep them separate, a little duplication is a good compromise.